src/lib/LUA/rx_devLUA.cpp

```diff
@@ -23,7 +23,7 @@ static const char *rxModes = "50Hz;60Hz;100Hz;160Hz;333Hz;400Hz;10kHzDuty;On/Off
 #endif
 
 static struct luaItem_selection luaSerialProtocol = {
-    {"Protocol", CRSF_TEXT_SELECTION},
+    {"Serial Protocol", CRSF_TEXT_SELECTION},
```

Comment:
That may be a little long for smaller display radios like the pocket etc

Reply:
![image](https://github.com/ExpressLRS/ExpressLRS/assets/5615068/28913018-d289-483b-a84f-cca17b7a119a)

I guess you are right. Looks like 11 characters max. How's "Serial Prot" look? Or just leave it as it is.

Reply:
Serial protocol does indeed get cut off.
![proto](https://github.com/ExpressLRS/ExpressLRS/assets/512740/3f0ddbfc-8b93-4eb6-832a-9325d7aa39a2)

Perhaps "Serial Comms" is better. I'm guessing you want to have "Serial" in the name so it is obvious that it's what is output on the serial pins.
![comms](https://github.com/ExpressLRS/ExpressLRS/assets/512740/48810ff7-16d4-49cd-b083-a23c86ab743d)

Reply:
Yes, but I also wanted the WebUI and LUA labels to be coherent

Reply:
> Yes, but I also wanted the WebUI and LUA labels to be consistent

Probably best to leave it as "Protocol", how I had it then 😄